rskj-core/src/main/java/co/rsk/peg/SerializationException.java

```diff
@@ -0,0 +1,16 @@
+package co.rsk.peg;
+
+
+public class SerializationException extends RuntimeException {
+    public SerializationException() {
+        super();
+    }
+
+    public SerializationException(String message) {
+        super(message);
+    }
```

Comment:
Do we really need these constructors? I think for logging consistency we should always use the one below.

Reply:
not sure, wdyt @marcos-iov ?

Reply:
True, if not needed then no reason to add it

Reply:
done